_data/anishbadhri/anishbadhri.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Anish Badri R S # No longer than 18 characters
+institution: SSN College of Engineering # no longer than 58 characters
+profile_pic: anishbadhri.png # Relative path to the image you uploaded
+quote: "It was a fun 4 years" # No longer than 100 characters
```

Comment:
Remove double quotes from begin and end.

Reply:
Done, thanks